src/webhook_launcher/processes/VCSCOMMIT_BUILD

```diff
@@ -30,6 +30,12 @@ Ruote.process_definition 'vcscommit' do
     # This should be conditional though
     create_project
     trigger_service
+    # We wait for the source before checking the build results as
+    # if the service is still running the is_repo_published can
+    # fail. This also allows source problems to be handled in
+    # other ways
```

Comment:
Not sure what to do with this last sentence.  Do you mean that having a separate do_wait_for_src step means that in the future one could add handling of its failure modes?

If so, I'd just leave that sentence off, it's more something you'd bring up in design/review than a code comment imo.